packages/clarity-js/src/core/scrub.ts

```diff
@@ -87,17 +88,22 @@ export function text(value: string, hint: string, privacy: Privacy, mangle: bool
     return value;
 }
 
-export function url(input: string, electron: boolean = false): string {
+export function url(input: string, electron: boolean = false, truncate: boolean = false): string {
+    let result = input;
     // Replace the URL for Electron apps so we don't send back file:/// URL
-    if (electron) { return `${Data.Constant.HTTPS}${Data.Constant.Electron}`; }
+    if (electron) { result = `${Data.Constant.HTTPS}${Data.Constant.Electron}`; }
 
     let drop = config.drop;
```

Comment:
looks like you need to wrap these non electron statements in an else now - we don't want them to execute when it's electron